### PR TITLE
it-15782 - Incorporated changes based on editorial review

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR incorporates the improvements suggested during the docathon editorial review.

### CODE REVIEW

File: downstream/assemblies/hub/assembly-configure-hub-primary.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ ifdef::context[:parent-context: {context}]
 :context: configure-hub
 
 [role="_abstract"]
-Configure Red Hat {HubName} as your primary source of content to access Ansible Certified Content Collections. You can configure {HubName} in the command-line interface (CLI) or the web console.
+to access Ansible Certified Content Collections, configure Red Hat {HubName} as your primary source of content. You can configure {HubName} in the command-line interface (CLI) or the web console.
```
Comment (hherbly) [inline downstream/assemblies/hub/assembly-configure-hub-primary.adoc:9]: I think this is the beginning of the sentence--if so, "to" should be capitalized.

File: downstream/assemblies/hub/assembly-hub-overview.adoc [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+{HubNameStart}:: Hosted by Red Hat, it contains only Red Hat supported or certified content.
+{PrivateHubNameStart}:: This is a self-hosted content management system. You can use it to access and manage all types of Ansible content and choose which Ansible content collections and versions are made available to your automation consumers.
 
-* Community content found in {Galaxy}
-* Red Hat certified and supported content found in {HubName} on the link:https://console.redhat.com/ansible/automation-hub[Red Hat Hybrid Cloud Console]
-* Private content created and curated by an organization and shared locally
+With a {PrivateHubName}, you can access these three types of content:
 
-The distinction between {HubName} and {PrivateHubName} is that {HubName} allows you to access only Red Hat supported or certified content, and {PrivateHubName}, as a self-hosted content management system, allows you to access and manage of all three types of content, choosing which Ansible content collections and versions are made available to your automation consumers.
+* Red Hat certified and supported content found in {HubName} on the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console]
```
Comment (hherbly) [inline downstream/assemblies/hub/assembly-hub-overview.adoc:16]: The sentences in these last three bullet points should end with periods for consistency with the first two bullets (and also bc they're complete sentences).

File: downstream/modules/hub/con-offline-token-active.adoc [hunk 3]
```diff
@@ -3,16 +3,19 @@
 
 = Keeping your offline token active
 
-Keeping an offline token active is useful when an application needs to perform an action on behalf of the user, like a routine data backup, even when the user is offline.
-
 Offline tokens expire after 30 days of inactivity. You can keep your offline token from expiring by periodically refreshing your offline token.
 
+Keeping an offline token active is useful when an application performs an action on behalf of the user; when the user is offline, such as a routine data backup.
```
Comment (hherbly) [inline downstream/modules/hub/con-offline-token-active.adoc:8]: I'm not sure I agree with this update. A semicolon should connect two complete clauses (i.e. they should be able to stand on their own); the second clause isn't complete. To keep the semicolon I would change to: 

Keeping an online token active is useful when an application performs an action on behalf of the user; for example, this allows the application to perform a routine data backup when the user is offline.

File: downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc [hunk 4]
```diff
@@ -1,14 +1,15 @@
 [id="proc-configure-automation-hub-server-cli"]
-= Configuring Red Hat automation hub as the primary source for content using the CLI
+= Using the CLI to configure Red Hat automation hub as the primary content source
 
-Configure Red Hat automation hub as your primary source of content by using the CLI. To configure automation hub, you must modify the `ansible.cfg` configuration file. By default, the `ansible.cfg` configuration file is located in the `/etc/ansible/` directory. With automation hub, you have access to certified, supported collections.
+To configure automation hub, you must modify the `ansible.cfg` configuration file. By default, the `ansible.cfg` configuration file is located in the `/etc/ansible/` directory. With automation hub, you have access to certified, supported collections.
```
Comment (hherbly) [inline downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc:4]: Missing attributes in lines 1-4 here.

File: downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc [hunk 5]
```diff
@@ -53,9 +57,11 @@ password=my_pass
...
 <3> Include the `/api/galaxy/` subdirectory in the {HubName} server URL.
 
-NOTE: All API URLs must end with a trailing slash */* in order to prevent receiving a 301 redirect.
-
-You have now configured automation hub as your primary server using CLI, and can proceed to download and install supported collections.
+[NOTE]
+====
+To prevent a 301 redirect, all API URLs must end with a trailing slash */*.
+====
+You have now configured automation hub as your primary server. You can begin to download and install supported collections.
```
Comment (hherbly) [inline downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc:64]: Missing attribute here.

**Review by hherbly — CHANGES_REQUESTED** [review]
I added a few small comments, but everything else looks good. Lmk if you have any questions.

File: downstream/assemblies/hub/assembly-configure-hub-primary.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/assemblies/hub/assembly-configure-hub-primary.adoc]: Good catch! Fixed.

File: downstream/modules/hub/con-offline-token-active.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/hub/con-offline-token-active.adoc]: Your suggestion is much better. This is fixed.

File: downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc]: Fixed.

File: downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc (hunk 5 above)
Comment (dcdacosta) [reply downstream/modules/hub/proc-configure-automation-hub-server-cli.adoc]: Fixed.

**Review by hherbly — APPROVED** [review]
LGTM!

**Pull request merged**